javaslang/src-gen/main/java/javaslang/Tuple0.java

```diff
@@ -127,12 +127,12 @@ private Object readResolve() {
     }
 
     @Override
-    public <T> Tuple1<T> prepend(T value) {
+    public <T> Tuple1<T> append(T value) {
```

Comment:
what was the problem with the previous order?
`prepend` comes conceptually before `append`, i.e. `insert before` should be before `insert after`, I think